File: BCKEND/JAVA/BancoFinanceiro/src/Model/Endereco.java
package Model;

public class Endereco {

	private String uf;
	private String cidade;
	private String bairro, rua, comp, cep;
	private int numero;

	public Endereco(String uf, String cidade, String bairro, String rua, String comp, String cep, int numero) {
		super();
		this.uf = uf;
		this.cidade = cidade;
		this.bairro = bairro;
		this.rua = rua;
		this.comp = comp;
		this.cep = cep;
		this.numero = numero;
	}

	public String getUf() {
		return uf;
	}

	public void setUf(String uf) {
		this.uf = uf;
	}

	public String getCidade() {
		return cidade;
	}

	public void setCidade(String cidade) {
		this.cidade = cidade;
	}

	public String getBairro() {
		return bairro;
	}

	public void setBairro(String bairro) {
		this.bairro = bairro;
	}

	public String getRua() {
		return rua;
	}

	public void setRua(String rua) {
		this.rua = rua;
	}

	public String getComp() {
		return comp;
	}

	public void setComp(String comp) {
		this.comp = comp;
	}

	public String getCep() {
		return cep;
	}

	public void setCep(String cep) {
		this.cep = cep;
	}

	public int getNumero() {
		return numero;
	}

	public void setNumero(int numero) {
		this.numero = numero;
	}

	@Override
	public String toString() {
		return "Endereco: \nUF= " + uf + "\nCidade= " + cidade + 
				"\nBairro= " + bairro + "\nRua= " + rua + 
				"\nNumero= " + numero +
				"\nComplemento= " + comp
				+ "\nCEP= " + cep ;
	}

}

File: BCKEND/TypeScript/BancoFinanceiro/src/Model/Endereco.ts
export class Endereco{

    private uf: string;
    private cidade: string;
    private bairro: string;
    private rua: string;
    private numero: number;
    private complemento?: string;
    private cep: string;

    constructor(uf: string, cidade: string, bairro: string, rua: string, numero: number, cep: string, complemento?: string){
        this.uf = uf;
        this.cidade = cidade;
        this.bairro = bairro;
        this.rua = rua;
        this.numero = numero;
        this.cep = cep;
        this.complemento = complemento ? complemento : "";
    }

    public getUF(): string{
        return this.uf;
    }

    public getCidade(): string{
        return this.cidade;
    }

    public getBairro(): string{
        return this.bairro;
    }

    public getRua(): string{
        return this.rua;
    }

    public getNumero(): number{
        return this.numero;
    }

    public getComplemento(): string | undefined{
        return this.complemento;
    }

    public getCep(): string{
        return this.cep;
    }

    public setUF(uf: string): void{
        this.uf = uf;
    }

    public setCidade(cidade: string): void{
        this.cidade = cidade;
    }

    public setBairro(bairro: string): void{
        this.bairro = bairro;
    }

    public setRua(rua: string): void{
        this.rua = rua;
    }

    public setNumero(numero: number): void{
        this.numero = numero;
    }

    public setComplemento(complemento: string): void{
        this.complemento = complemento;
    }
    
    public setCep(cep: string): void{
        this.cep = cep;
    }

    public toString(): string{
        return `${this.rua}, ${this.numero} - ${this.bairro}, ${this.cidade} - ${this.uf}, CEP: ${this.cep}` + (this.complemento ? `, Complemento: ${this.complemento}` : '');
    }

}